nativeApp/src/lib/ui/forms/input.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components/native'
+import { Fonts } from '../typography';
+import { ColoursEnum } from '../colours';
+
+const InputContainer = styled.View`
+  flexDirection: row;
```

Comment:
how come all these are in camel case instead of kebab case?

Reply:
see [previous comment](https://github.com/TwinePlatform/twine-monolith/pull/329#discussion_r329993833)

Reply:
ok but we're still using kebab case [here](https://github.com/TwinePlatform/twine-monolith/pull/329/files#diff-6e47240cf4c2dc353fa0e55cbf7750d4), should that be made consistent?

Reply:
Oops, yeah that's a mistake